scopes/react/component-highlighter/label/label.module.scss

```diff
@@ -1,3 +1,3 @@
 .label {
-  font-size: 12px;
+  font-size: .75em;
```

Comment:
are you sure about this? wont it make the labels bigger when hovering on large font sizes?

Reply:
yes. The label size is coming from the highlighter font size.
I guess I could set it to be 12px, and then it would be easy to override